test/suite/test_checkpoint_snapshot05.py

```diff
@@ -147,11 +147,11 @@ def test_checkpoint_snapshot(self):
         try:
             ckpt.start()
             
-            # Wait for checkpoint to start before committing.
+            # Wait for checkpoint to start and acquire its snapshot before committing.
             ckpt_started = 0
```

Comment:
how about changing the variable name to `ckpt_snapshot` or similar because we are waiting until the checkpoint has acquired the snapshot?

Reply:
Fixed in [07a0be7](https://github.com/wiredtiger/wiredtiger/pull/9063/commits/07a0be746803804f00856fb19b57680bba824969)